### PR TITLE
Hide Organization Edit Pencils by default

### PR DESCRIPTION
**Pull request by Devorein**

### CODE REVIEW

File: components/profile/components/SpacesMemberDetails/components/SpaceDetailsAccordion.tsx [hunk 1]
```diff
@@ -16,9 +18,28 @@ type Props = {
   expanded?: boolean;
 };
 
+const StyledAccordionSummary = styled(AccordionSummary)`
+  ${({ theme }) => !isTouchScreen() && `
+    &:hover .icons {
+      opacity: 1;
+      transition: ${theme.transitions.create('opacity', {
+    easing: theme.transitions.easing.easeInOut,
+    duration: theme.transitions.duration.enteringScreen
```
Comment (msadura) [inline components/profile/components/SpacesMemberDetails/components/SpaceDetailsAccordion.tsx:27]: formatting looks really weird, but this is just a tiny thing

File: components/profile/components/UserDetails/UserDetails.tsx [hunk 2]
```diff
@@ -49,6 +51,41 @@ export interface UserDetailsProps {
   sx?: SxProps;
 }
 
+const StyledStack = styled(Stack)`
+  &:hover .icons {
```
Comment (msadura) [inline components/profile/components/UserDetails/UserDetails.tsx:55]: looks like it does pretty much the same as styles Accordion. Moreover we will most likely  use this behavior in other places. I would place those in separate file like `hoverIconsStyle` export to reuse with emotion `css` . Then you can import such css and reuse in every place you need it

**Review by msadura — CHANGES_REQUESTED** [review]

File: components/profile/components/SpacesMemberDetails/components/SpaceDetailsAccordion.tsx (hunk 1 above)
Comment (Devorein) [reply components/profile/components/SpacesMemberDetails/components/SpaceDetailsAccordion.tsx:27]: yes tried a lot to fix this but as soon as the file is saved, vscode formats it wildly

File: components/profile/components/UserDetails/UserDetails.tsx (hunk 2 above)
Comment (Devorein) [reply components/profile/components/UserDetails/UserDetails.tsx]: Agree. Create a separate file, exported the `hoverIconsStyle` constant and reused in the places its used

**Review by msadura — APPROVED** [review]

**Pull request merged**